packages/playground/src/components/layout/top-bar-widget.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import {
+  Topbar,
+  Breadcrumb,
+  BreadcrumbList,
+  BreadcrumbItem,
+  DropdownMenu,
+  DropdownMenuTrigger,
+  BreadcrumbLink,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  Icon,
+  Avatar,
+  AvatarFallback,
+  BreadcrumbSeparator,
+  cn
+} from '@harnessio/canary'
+import { useParams } from 'react-router-dom'
+
+// Project Dropdown Component
+const AvatarDropdown = ({ isPrimary }: { isPrimary: boolean }) => (
+  <DropdownMenu>
+    <DropdownMenuTrigger className="group flex items-center gap-2 outline-none">
+      <Avatar className="w-7 h-7">
+        <AvatarFallback>P</AvatarFallback>
```

Comment:
playground components should NOT have hardcoded data in them. please add the correct props to make the component actually useable outside the playground